orbitale/easyimpress-bundle/1.0/post-install.txt

```diff
@@ -0,0 +1,4 @@
+<bg=blue;fg=white> Next for EasyImpressBundle </>
+    Create your presentations in yaml in the <comment>config/presentations/</comment> directory.
+
+    Check documentation for yaml reference options: <comment>https://github.com/Orbitale/EasyImpressBundle#yml-reference</comment>.
```

Comment:
This file should be removed. People probably come from the docs anyway. We try to limit the usage of the post install messages to when it's really important, like having to run a command before getting started.

Reply:
Removed